examples/calendar/backend/services/time/controller.ts

```diff
@@ -0,0 +1,92 @@
+import { NextFunction, Request, Response } from "express";
+//import { User } from "../../models/user.model";
+import { Time } from "../../models/time.model";
+import { Op } from "sequelize";
+
+// Create time availablity for the user
+export const create = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  if ((req as any).user.payload.id !== +req.body.userId) {
+    return res.status(401).send({ error: "You can can only access yourself" });
+  }
+  const { timeStarts, timeEnds, userId } = req.body;
+  if (timeStarts.length !== timeEnds.length) {
+    return res
+      .status(400)
+      .send({ error: "Something went wrong! Please contact developer" });
```

Comment:
`Please contact developer` doesn't seem to be correct here.

Reply:
This has been updated.